README.md

```diff
@@ -1,833 +1,477 @@
 
-# skyflow-node
-skyflow-node is the Node.js version of Skyflow SDK for the JavaScript programming language.
+# Skyflow Node.js SDK
+
+SDK for the Skyflow Data Privacy Vault.
 
 [![CI](https://img.shields.io/static/v1?label=CI&message=passing&color=green?style=plastic&logo=github)](https://github.com/skyflowapi/skyflow-node/actions)
 [![GitHub release](https://badge.fury.io/js/skyflow-node.svg)](https://www.npmjs.com/package/skyflow-node)
 [![License](https://img.shields.io/github/license/skyflowapi/skyflow-node)](https://github.com/skyflowapi/skyflow-node/blob/master/LICENSE)
 
 
-# Table of Contents
+## Table of Contents
 
-- [skyflow-node](#skyflow-node)
-- [Table of Contents](#table-of-contents)
+- [Skyflow Node.js SDK](#skyflow-nodejs-sdk)
+  - [Table of Contents](#table-of-contents)
   - [Installation](#installation)
-    - [Requirements](#requirements)
-    - [Configuration](#configuration)
-  - [Usage](#usage)
-    - [Importing `skyflow-node`](#importing-skyflow-node)
-    - [Service Account Bearer Token Generation](#service-account-bearer-token-generation)
-    - [Service Account Bearer Token Generation with Additional Context](#service-account-bearer-token-generation-with-additional-context)
-    - [Service Account Scoped Bearer Token Generation](#service-account-scoped-bearer-token-generation)
-    - [Skyflow Signed Data Tokens Generation](#skyflow-signed-data-tokens-generation)
-    - [Vault APIs](#vault-apis)
-      - [Insert](#insert)
-      - [Detokenize](#detokenize)
-      - [Get By Id](#get-by-id)
-      - [Get](#get)
-      - [Update](#update)
+      - [Requirements](#requirements)
+      - [Import / Require](#import--require)
+  - [Vault APIs](#vault-apis)
+    - [Insert: vault.insert()](#insert-vaultinsert)
+      - [Example: Insert Records](#example-insert-records)
+      - [Example: Upsert Records (update or insert)](#example-upsert-records-update-or-insert)
+      - [Example: Insert with partial success support](#example-insert-with-partial-success-support)
+    - [Detokenize: vault.detokenize()](#detokenize-vaultdetokenize)
+      - [Detokenize example](#detokenize-example)
+    - [Get By IDs: vault.getById()](#get-by-ids-vaultgetbyid)
+      - [Example: Get Records by IDs](#example-get-records-by-ids)
+    - [Get records by unique values: vault.get()](#get-records-by-unique-values-vaultget)
+      - [Example: Get records by ID(s)](#example-get-records-by-ids-1)
+      - [Example: Get records by unique field values](#example-get-records-by-unique-field-values)
+    - [Update records by IDs](#update-records-by-ids)
+      - [Example: Update records by IDs](#example-update-records-by-ids)
       - [Delete](#delete)
```

Comment:
TODO: I probably need to elevate this heading one level and update it to match the rest.